news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,127 @@
+
+This meant that 3 misses on a 100 note map was functionally equivalent to 3 misses on a 1000 note map.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- The miss penalty curve has been adjusted.
+    - Any number of misses on a map instantly incurs a once-off 3% penalty to total pp awarded for both aim and speed.
+    - The first few misses are penalised less harshly than the ones after. The largest relative punishment occurs somewhere around the 15% miss mark (relative to the number of objects in the map), after which point the reduction for each subsequent miss tapers off again.
+    - Old value: Aim and speed values reduced by ```0.97^x``` where x equals the number of misses.
+
+To help illustrate these changes better, consult the graph image below. The green line refers to the old formula, and the purple line refers to the new one. You can also view this graph live and play around with the settings following [this link](https://www.desmos.com/calculator/rshij9757a).
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/miss-penalty-curve.png)
+
+This change seeks to penalize misses on short maps more, while significantly reducing the impact of a small number of misses on longer maps. The one-off 3% penalty for a miss provides incentive for players to hone any 1-miss plays they might still make, but otherwise makes them considerably less punishing on longer plays.
+
+## Speed & Accuracy Curve Adjustments
+
+As osu! has evolved over time, accuracy has slowly but steadily fallen to the wayside as a principle component for determining the pp values of modern scores. To illustrate this, 500-700pp scores from many years ago sometimes derived as much as 20-40% of their overall pp purely from the accuracy component alone.
+
+Today, many scores pull less than 20% of their score from accuracy instead, relying on raw speed as the largest influencing component on how scores are set. This has reduced the meaningful impact of accuracy as a skill considerably, which is a peculiar approach for a rhythm game where accuracy should be king.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- A new speed curve has been introduced which factors in Overall Difficulty and accuracy.
+    - New value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/new-speed-curve.png)
+    - Old value: ![](/wiki/shared/news/2021-01-14-performance-points-updates/old-speed-curve.png)
+- The new speed curve, in opposition to the previous one, is no longer linear with respect to accuracy, opting for an exponential instead. This severely nerfs low-accuracy scores, especially around the 60% accuracy mark.
```

Comment:
```suggestion
- The new speed curve, in opposition to the previous one, is no longer linear with respect to the accuracy, opting for an exponential instead. This severely nerfs low-accuracy scores, especially around the 60% accuracy mark.
```
lack of determiner

Reply:
if you add "the" here you also need to make it "the accuracy of a play" or something. fine as-is.